site/app/models/gradeable/GradeableList.php

```diff
@@ -312,10 +312,10 @@ public static function getGradeableSection(Core $core, Gradeable $gradeable): in
         ) {
             return self::OPEN;
         }
-        elseif ($core->getUser()->accessGrading() && $gradeable->getTaViewStartDate() <= $now) {
+        elseif ($gradeable->getTaViewStartDate() <= $now) {
             return self::BETA;
         }
-        elseif ($core->getUser()->accessAdmin()) {
+        else {
```

Comment:
maybe an elseif here is better instead of an else just in case

Reply:
alright, I rewrote that specific elseif and pushed the change
you may review.